src/styles/card.scss

```diff
@@ -0,0 +1,155 @@
+@import './variables';
+@import './mixins';
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @include on-mobile {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @include on-tablet {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @include on-desktop {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  width: 198px;
+  height: 404px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 0.3s ease-in-out;
+
+  &__photo {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: $big-line-height;
+    margin: 40px 16px 0;
+
+    & > p {
+      margin: 0 0 4px;
+    }
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__subtitle {
+    color: $secondary-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: $default-line-height;
+    margin: 0 16px;
+    & > p {
+      margin: 0;
+    }
+  }
+
+  &__feedback {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin: 16px 16px 24px;
+
+    &-review {
+      display: flex;
+      color: $main-accent;
+      font-size: 10px;
+      font-weight: 400;
```

Comment:
Personally I wouldn't recommend this nesting, as `__feedback` and `__feedback-review` although share part of the name are two distinct elements.

**This is a personal suggestion and not a required change**

Reply:
It makes sense actually, gonna change that!